templates/Includes/FlexSlider.ss

```diff
@@ -18,17 +17,20 @@
             </ul>
         </div>
         <% if $ThumbnailNav && $SlideShow.Count > 1 %>
-        <div class="fs-carousel">
+        <div class="fs-carousel col-md-12">
             <ul class="slides">
                 <% loop $SlideShow %>
                     <li>
                         <% if $Image %>
-                            <img src="$Image.URL"  alt="$Image.Title" class="slide">
+                            <img src="$Image.Fill(200,200).URL"  alt="$Image.Title" class="slide">
                         <% end_if %>
                     </li>
                 <% end_loop %>
             </ul>
         </div>
     <% end_if %>
-</div>
+    </div>
+    <% require javascript('//code.jquery.com/jquery-3.6.1.min.js') %>
```

Comment:
Based on our convo, if we can localize this and not call it from the CDN it could then be blocked by developers if they bring their own version.

Reply:
For now just copying it to a thridparty directory is fine, we can look at FE workflows in a later commit.